diagnostics/stc_annular_modes/stc_annular_modes_plot.py

```diff
@@ -0,0 +1,250 @@
+# ==============================================================================
+# MDTF Strat-Trop Coupling: Annular Modes POD
+# ==============================================================================
+#
+# This file is part of the Strat-Trop Coupling: Annular Modes POD of the MDTF 
+# code package (see mdtf/MDTF-diagnostics/LICENSE.txt). It defines the plotting 
+# functions used by the main driver script.
+
+import numpy as np
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+
+mpl.rcParams["font.family"] = "sans-serif"
+mpl.rcParams["font.sans-serif"] = "Roboto"
```

Comment:
This will just default to the DejaVu Sans (or something else) if the user does not have this font on their system, correct?

Reply:
Yes, this is correct.